tracer/contrib/go-redis/tracedredis.go

```diff
@@ -0,0 +1,146 @@
+package tracedredis
+
+import (
+	"bytes"
+	"context"
+	"github.com/DataDog/dd-trace-go/tracer"
+	"github.com/DataDog/dd-trace-go/tracer/ext"
+	"github.com/go-redis/redis"
+	"strconv"
+	"strings"
+)
+
+// TracedClient is used to trace requests to a redis server.
+type TracedClient struct {
+	*redis.Client
+	traceParams TraceParams
+}
+
+// TracedPipeline is used to trace pipelines with a redis server.
+type TracedPipeline struct {
+	*redis.Pipeline
+	traceParams TraceParams
+}
+
+// TraceParams contains the tracer and params that we want to trace.
+type TraceParams struct {
+	host    string
+	port    string
+	db      string
+	service string
+	tracer  *tracer.Tracer
+}
+
+// NewTracedClient needs to be called instead of NewClient to trace the calls.
+func NewTracedClient(opt *redis.Options, t *tracer.Tracer, service string) *TracedClient {
+	var host, port string
+	addr := strings.Split(opt.Addr, ":")
+	host = addr[0]
+	port = addr[1]
```

Comment:
Same thing as in #51 double-check for the number of items and use default port if not given.